app/scripts/directives/unbindService.js

```diff
@@ -26,11 +26,13 @@
     var serviceInstanceDisplayName = $filter('serviceInstanceDisplayName');
 
     var unbindService = function() {
+      ctrl.deletedBinding = _.first(_.filter(ctrl.bindings, {metadata: {name: ctrl.selectedBinding}}));
+      ctrl.appsForDeletedBinding = angular.copy(ctrl.appsForBinding(ctrl.selectedBinding));
```

Comment:
right, I think thats why I'm copying it.  Otherwise the related apps don't map by the time the results panel appears.